src/layouts/Sites.jsx

```diff
@@ -22,6 +22,12 @@ export default class Sites extends Component {
         withCredentials: true,
       });
       const { siteNames } = resp.data;
+
+      window.sitesIsPrivate = siteNames.reduce((map, siteNames) => {
+        map[siteNames.repoName] = siteNames.isPrivate
+        return map
+      })
```

Comment:
I took a look at the value of `window.sitesIsPrivate` in the browser console and discovered that there were some unexpected fields in the map. We see `lastUpdated`, `permissions`, `repoName`, `isPrivate` as fields of the map, when we should only be seeing `<repoName>` as the fields.

Additionally, I couldn't find the value that corresponded to the `a-test` repo.

<img width="747" alt="Screenshot 2021-05-21 at 4 11 51 PM" src="https://user-images.githubusercontent.com/19917616/119104862-55d2e780-ba4f-11eb-9829-86a16b893a59.png">

The main reason this is happening is because we currently don't provide the `Array.reduce` method with an `initialValue`. And if we don't provide `initialValue`, `the first element in the array will be used as the initial accumulator value and skipped as currentValue` [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce).

As a result, `map` is instantialized as 
```
{
  "lastUpdated": "Updated 1 day ago",
  "permissions": {
      "admin": true,
      "push": true,
      "pull": true
  },
  "repoName": "a-test",
  "isPrivate": false
}
```

And subsequent iterations will continue adding to `map` as such
```
{
  "lastUpdated": "Updated 1 day ago",
  "permissions": {
      "admin": true,
      "push": true,
      "pull": true
  },
  "repoName": "a-test",
  "isPrivate": false,
  "a-test-v4": "true",
  // ... rest of sites
}
```

The fix is to provide the `reduce` function with an `initialValue` of `{}`, i.e. line 29 should be:
```
}, {})
```